it/mail/chrome/messenger/news.properties

```diff
@@ -36,55 +36,55 @@
 # LOCALIZATION NOTE ( htmlNewsError ): In the following item, translate only "Error!" and "newsgroup server responded:"
-htmlNewsError=<H1>Errore!</H1> Il server dei gruppi discussione ha risposto: 
+htmlNewsError=<H1>Errore!</H1> Il server newsgroup ha risposto:
 # LOCALIZATION NOTE ( articleExpired ): In the following item, translate only "Perhaps the article has expired"
 articleExpired=<B><P>Probabilmente l'articolo è scaduto</B><P>
-removeExpiredArtLinkText=Fare clic qui per rimuovere tutti gli articoli scaduti
+removeExpiredArtLinkText=Clicca qui per rimuovere tutti gli articoli scaduti
 cancelDisallowed=Questo messaggio sembra non appartenere a te. È possibile eliminare unicamente i propri messaggi, non quelli creati da altri.
 cancelConfirm=Eliminare questo messaggio?
 messageCancelled=Messaggio eliminato.
-enterUsername=Inserire un nome utente per l'accesso al server delle news:
-enterUsernameTitle=News Server Username Required
-saveUsername=Use Password Manager to remember this value.
-enterPassword=Inserire una password per l'accesso al server delle news:
+enterUsername=Inserisci un nome utente per l'accesso al server delle news:
+enterUsernameTitle=Nome utente server news richiesto
+saveUsername=Utilizza il Gestore password per ricordare questo valore.
+enterPassword=Inserisci una password per l'accesso al server delle news:
 enterPasswordTitle=Password server delle news
 okButtonText=Scarica
 
 noNewMessages=Non ci sono nuovi messaggi sul server.
-downloadingHeaders=Downloading %S of %S headers
-downloadingArticles=Ricezione articoli %S-%S
-bytesReceived=Ricezione gruppi disc.: ricevuti %S (%SkB letti a %SkB/s)
-checkingForNewNews=Checking newsgroup %S of %S on %S for new messages
-downloadingArticlesForOffline=Ricezione articoli %S-%S in %S
+downloadingHeaders=Download %S di %S intestazioni
+downloadingArticles=Download articoli %S-%S
+bytesReceived=Download gruppi disc.: ricevuti %S (%SkB letti a %SkB/s)
+checkingForNewNews=Controllo newsgroup %S di %S su %S per nuovi messaggi in corso
+downloadingArticlesForOffline=Download articoli %S-%S in %S in corso
 
-onlyCancelOneMessage=È possibile annullare solo un articolo per volta.
+onlyCancelOneMessage=Puoi annullare solo un articolo per volta.
 
 # LOCALIZATION NOTE (autoUnsubscribeText): %1$S is the newsgroup and %2$S is the newsgroup-server it is being removed from.
-autoUnsubscribeText=Il gruppo %1$S non sembra esistere sull'host %2$S. Rimuovere la sottoscrizione?
+autoUnsubscribeText=Il gruppo %1$S non sembra esistere sull'host %2$S. Vuoi disiscriverti?
 
 # LOCALIZATION NOTE (autoSubscribeText): %1$S is the newsgroup.
-autoSubscribeText=Sottoscrivere %1$S?
+autoSubscribeText=Vuoi iscriverti a %1$S?
 
 # LOCALIZATION NOTE (Error -304): In the following item, don't translate "NNTP"
 # Error - server error
 ## @name NNTP_ERROR_MESSAGE
 ## @loc None
--304=Si è verificato un errore con i gruppi di discussione (NNTP):  
+-304=Si è verificato un errore con le news (NNTP):
```

Comment:
I would say "Si è verificato un errore con le news (NNTP):". In the source text, it is specified "news" and not "newsgroup".